### DISCUSSION TITLE
Do I need to create a post folder?

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#289] [Q&A] [answered]
I'm working on the task to upload a portfolio post and this is what my directory looks like:
![image](https://github.com/user-attachments/assets/176379c2-394e-4401-9151-496c33c5b1a1)

The video directory looks like this, so I'm thinking the step of creating a folder was overlooked because I went through all the steps, but my first-map did not show up in my directory.
![image](https://github.com/user-attachments/assets/cd735f00-6192-455f-be96-5d457ec63d21)

**Comment by nquarder** [comment, marked as answer]
Hi @BioNCphilic and great question. The goal for this part of the assignment is to add your first-map contributions to the posts folder in the Earth-Analytics-AY24 course repository. You'll do this by first making edits to your `username-contrib` branch and then submitting a pull request. 

If your `username-contrib` branch is current with the main branch, there shouldn't be a need for you to create a posts folder. If you go back and watch @eculler 's video, you'll notice that she demonstrates how to do this, though it might be easy to miss as she went through it pretty quickly! ;) 

We'll have time during our discussion section today (Weds 1-2:30pm MST) to go through this again. 

Once your branch is current, from there you should be able to create a folder with your username in the `posts/01-first-map` directory to add your portfolio posts. Be sure to create a .keep file when making the folder (@eculler show's how to do this too in her video!). Once you've added those files (.html, .md) to your branch, submit a pull request so that your work shows up in the main branch. 

Again, we should have some time to demonstrate how to do this in our discussion section today. -NQ
